### ISSUE TITLE
*boss* thread preventing SBCL from dumping images

### ISSUE THREAD

**Issue by zkat** [zkat/chanl#7]
I got this via email -- posting it here:

I have a little project of mine in common lisp, and I tried to run it on a "new" computer an hour ago.
This being a fresh install, it used the latest release of chanl from quicklisp.
But my build failed because it could not save a core image.

(P.S. I am using sbcl from HEAD)

The problem is that sbcl cannot save core images that have multiple running threads.
I dug the problem up and I found that it was chanl that was creating the superfluous thread.
I dug up a little bit more and I found that the thread was created at the initialization of the *boss* special variable.

Sooooo, is there a way that I can post-pone the initialization of the *boss* variable?

TL;DR:

The thread created for *boss* is causing me pain, what can I do for that?

**Comment by adlai** [comment]
The immediate fix is `chanl.actors::(halt *boss*)`, which will let you save the image, and shouldn't affect the rest of the library if you're not relying on this facility.

A proper fix will be pushed momentarily.

**Comment by mpsyco** [comment]
It worked like a charm. Thanks! (I was the one who send that email.)
